joern-cli/frontends/ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/Ghidra2Cpg.scala

```diff
@@ -189,6 +190,9 @@ class Ghidra2Cpg(
         }
     }
 
+    if (proc == "MIPS") {
```

Comment:
There is a match case for "MIPS". Why do you rewrite it and add an additional if statement?

Reply:
upsi, yes, that doesn't make sense, updated